tools/applicationinsights-web-snippet/src/snippet.ts

```diff
@@ -234,7 +233,7 @@ declare var cfg:ISnippetConfig;
                         if (xhr.status === 200) {
                             var json = JSON.parse(xhr.responseText);
                             integrity = json.ext[targetType].integrity;
-                            targetSrc = match[1] + json.ext[targetType].file.substring(3);
+                            targetSrc = match[1] + json.ext[targetType].file;
```

Comment:
Actually, we should add some additional checks here so we can correctly handle any error cases like
- Unable to convert the response to JSON (this will either throw or return an empty object)
- Check for the existence of the `json.ext` `json.ext[targetType]` as these would case an exception to be thrown

A try/catch might be useful for these rather than testing for all cases, so that if for some reason we fail to fetch/process the json response we still fall back to the error case of either sending an "SDK Load failure" message (not a console log) or just trying to load the default provided version without the integrity attribute